.github/workflows/publish_pytorch_dev_docker.yml

```diff
@@ -71,4 +84,54 @@ jobs:
             AMDGPU_TARGETS=${{ matrix.targets.amdgpu_target }}
           push: true
           tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
+          labels: |
+            ${{ steps.meta.outputs.labels }}
+            org.opencontainers.image.title="PyTorch ROCm Dev Image"
+            org.opencontainers.image.description="Nightly ROCm PyTorch Dev Docker for target ${{ matrix.targets.amdgpu_target }}""
+            org.opencontainers.image.version=${{ env.VERSION }}
+            org.opencontainers.image.created=${{ steps.meta.outputs.created }}
+            org.opencontainers.image.revision=${{ github.sha }}
+
+      - name: Log digest
+        run: |
+          echo "Built image with digest: ${{ steps.build_push.outputs.digest }}"
+
+      - name: Trigger verification workflow
+        uses: benc-uk/workflow-dispatch@e2e5e9a103e331dad343f381a29e654aea3cf8fc #1.2.4
+        with:
+          workflow: verify_docker_image.yml
+          inputs: |
+            {
+              "amdgpu_target": "${{ matrix.targets.amdgpu_target }}",
+              "image_digest": "${{ steps.build_push.outputs.digest }}",
+              "runner_type": "${{
+                matrix.targets.amdgpu_target == 'gfx942' && 'linux-mi300-1gpu-ossci-rocm' ||
+                (matrix.targets.amdgpu_target == 'gfx1201' && 'linux-rx9070-gpu-rocm') ||
+                (matrix.targets.amdgpu_target == 'gfx1100' && 'linux-rx7900-gpu-rocm') ||
+                'linux-mi300-1gpu-ossci-rocm'
+              }}",
+              "dispatch_id": "${{ env.DISPATCH_ID }}"
+            }
+
+      - name: Wait for verification to complete
+        run: |
+          external-builds/pytorch/wait_for_verification.sh
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          DISPATCH_ID: ${{ env.DISPATCH_ID }}
+          START_TIME: ${{ env.START_TIME }}
+          GITHUB_REPOSITORY: ${{ github.repository }}
+
+      # Tag and push verified image
+      - name: Tag and push verified image
+        run: |
+          IMAGE=ghcr.io/rocm/therock_pytorch_dev_ubuntu_24_04_${{ matrix.targets.amdgpu_target }}
+          DIGEST=${{ steps.build_push.outputs.digest }}
+
+          echo "Using image: $IMAGE"
+          echo "Using digest: $DIGEST"
+          echo "Using version: $VERSION"
+
+          docker pull $IMAGE@$DIGEST
+          docker tag $IMAGE@$DIGEST $IMAGE:$VERSION
```

Comment:
Is it worth to use docker/build-push-action (https://docs.docker.com/build/ci/github-actions/manage-tags-labels/) here?

Reply:
I did not use` docker/build-push-action` because in this phase we are not building the image from scratch. We just pull and tag it. Is there anything I am missing here that will benefit us to use  `docker/build-push-action  `